src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
Can the cart service call `this.cartService.giftStartDate` and map the cart line items to include it? I'm a little confused why sometimes we are using `gift-start` and sometimes have to go compute it from `recurring-day-of-month`.